From f94cad82d17189de3145505e0eb0591274b3b578 Mon Sep 17 00:00:00 2001
From: Suresh Gupta <suresh.gupta@xilinx.com>
Date: Wed, 6 Feb 2019 19:31:44 +0530
Subject: [PATCH 0504/1851] documentation: dt: multi-scaler: Add properties for
 pixels per clock

commit b7f6c0cff38ccf61ffe8c675bebb8e9587607ecc from
https://github.com/Xilinx/linux-xlnx.git

Adds the pixels per clock device tree property.

By introducing these changes to the DT properties, the backward
compatibility is broken between DT and driver. So, it is mandatory to
take new DT with latest driver.

Signed-off-by: Suresh Gupta <suresh.gupta@xilinx.com>
Reviewed-by: Satish Kumar Nagireddy <satish.nagireddy.nagireddy@xilinx.com>
Reviewed-by: Vishal Sagar <vishal.sagar@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/media/xilinx/xlnx,v-multi-scaler.txt    | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,v-multi-scaler.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-multi-scaler.txt
index 969c72e94513..6e1b55f12d8a 100644
--- a/Documentation/devicetree/bindings/media/xilinx/xlnx,v-multi-scaler.txt
+++ b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-multi-scaler.txt
@@ -12,6 +12,7 @@ Required propertie(s):
 - xlnx,max-width	: Maximum number of supported column/width (64 - 3840)
 - xlnx,max-height	: Maximum number of supported row/height (64 - 2160)
 - xlnx,dma-addr-width	: dma address width (either 32 or 64)
+- xlnx,pixels-per-clock	: pixels per clock set in IP (1, 2 or 4)
 - xlnx,vid-formats	: A list of strings indicating what video memory
 			  formats the IP has been configured to support.
 			  See VIDEO FORMATS table below and examples.
@@ -85,6 +86,7 @@ v_multi_scaler_0: v_multi_scaler@a0000000 {
 	reset-gpios = <&gpio 78 1>;
 	xlnx,max-chan = <0x01>;
 	xlnx,dma-addr-width = <0x20>;
+	xlnx,pixels-per-clock = /bits/ 8 <2>;
 	xlnx,max-width = <3840>;
 	xlnx,max-height = <2160>;
 	xlnx,num-taps = <6>;
-- 
2.31.1

